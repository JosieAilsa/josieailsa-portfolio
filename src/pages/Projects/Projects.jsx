const Projects = () => {
    return (
        <main>


        </main>
    );
}
 
export default Projects;