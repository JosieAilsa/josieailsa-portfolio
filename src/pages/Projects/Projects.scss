@use "../../assets/sass/abstracts/mixin" as *;
@use "../../assets/sass/abstracts/variables" as *;
@use "../../assets/sass/base/typography" as *;

.projects {
  .projects__heading {
    color: $color-green;
    grid-area: 1/3/1/12;
  }
}
