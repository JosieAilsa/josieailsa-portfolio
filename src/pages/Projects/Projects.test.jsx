import { customRender } from "../../utils/testutils";
import Projects from "./Projects";

test("Projects page should render", () => {
    const { container } = customRender(<Projects/>);
    expect(container).toMatchSnapshot();
  });
  