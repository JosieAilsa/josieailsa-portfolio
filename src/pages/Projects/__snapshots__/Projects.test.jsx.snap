// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects page should render 1`] = `
<div>
  <div
    class="page"
    id="projects"
  >
    <h3
      class="projects__heading  page__heading"
    >
      My Projects
    </h3>
    <section
      class="carousel"
    >
      <div
        class="carousel__image-container"
      >
        <img
          alt=""
          class="carousel__flower carousel__flower--petals"
          src="flowers-07.svg"
        />
        <img
          alt=""
          class="carousel__flower carousel__flower--pink"
          src="flowers-01.svg"
        />
        <img
          alt=""
          class="carousel__flower carousel__flower--green"
          src="flowers-09.svg"
        />
        <div
          class="carousel-item__image-wrap carousel-item__image-wrap--main"
        >
          <img
            alt="Save Oom"
            class="carousel-item__image carousel-item__image--main"
            src="save-oom.png"
          />
        </div>
        <div
          class="carousel-item__image-wrap carousel-item__image-wrap--two"
        >
          <img
            alt="Save Oom"
            class="carousel-item__image carousel-item__image--two"
            src="java.png"
          />
        </div>
        <div
          class="carousel-item__image-wrap carousel-item__image-wrap--three"
        >
          <img
            alt="Save Oom"
            class="carousel-item__image carousel-item__image--three"
            src="portfolio.png"
          />
        </div>
        <div
          class="carousel-item__image-wrap carousel-item__image-wrap--four"
        >
          <img
            alt="Save Oom"
            class="carousel-item__image carousel-item__image--four"
            src="save-oom.png"
          />
        </div>
      </div>
      <div
        class="carousel__buttons"
      >
        <button
          class="carousel__button carousel__button-left carousel__button--inactive"
        >
          <img
            alt=""
            class="carousel__button-image"
            data-id="click-left"
            src="arrow-left.svg"
          />
        </button>
        <button
          class="carousel__button carousel__button-right"
        >
          <img
            alt=""
            class="carousel__button-image"
            data-id="click-right"
            src="arrow-right.svg"
          />
        </button>
      </div>
      <article
        class="carousel__copy"
      >
        <h4
          class="carousel__header"
        >
          1. Save Oom
        </h4>
        <p
          class="carousel__text"
        >
          The first client project I lead with a cohort of students. I guided the Product Owner through reviewing PRs, requesting changes and merge conflicts. I also added the more complex sections of the project - including adding the user context, more complex RTL tests and Firebase integrations.
        </p>
        <div
          class="carousel__code-container"
        >
          <button
            class="button--secondary"
          >
            <a>
              See my code
            </a>
          </button>
          <button
            class="button--secondary"
          >
            <a
              href="https://save-oom.web.app/"
            >
              See the site
            </a>
          </button>
        </div>
      </article>
    </section>
  </div>
</div>
`;
